packages/data-driver-postgres/src/query/from.ts

```diff
@@ -1,11 +1,14 @@
 import type { AbstractSqlClauses } from '@directus/data-sql';
 import { escapeIdentifier } from '../utils/escape-identifier.js';
+import { tableIndexToIdentifier } from '../utils/index-to-identifier.js';
 
 /**
- * Generate the `FROM x` part of a SQL statement
- * @param from - The table to select data from
+ * Generate the `FROM x` part of a SQL statement.
+ *
+ * @param query - The whole abstract SQL query
  * @returns The `FROM x` part of a SQL statement
  */
 export function from({ from }: AbstractSqlClauses): string {
-	return `FROM ${escapeIdentifier(from)}`;
+	const tableAlias = tableIndexToIdentifier(from.tableIndex);
```

Comment:
It seems to me that we can create the identifier within the conversion step regardless of the driver since is should be dialect independent 🤔